kie-api/src/main/java/org/kie/api/task/TaskContext.java

```diff
@@ -15,9 +15,21 @@
  */
 package org.kie.api.task;
 
+import org.kie.api.task.model.Task;
 
 public interface TaskContext {
 
+    /**
+     * Returns currently configured UserGroupCallback.
+     * @return
```

Comment:
This will generate javadoc warning (missing value for @return tag). Please remove it or add some additional description.